### PR TITLE
Story/ft2 835 banner content picker

### PR DESCRIPTION
**Pull request by pramireddy**
Updated PageBanner ContentType
Added new controller BannerContentPickerAdminController.cs
Added new Razor views for Banner content option
Added new Service BannerContentPickerResultProvider.cs

### CODE REVIEW

File: DFC.ServiceTaxonomy.ContentPickerPreview/Views/ContentPickerField-Banner.Edit.cshtml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    var bannerContentItems = await Session.Query<ContentItem, ContentItemIndex>().Where(x => x.ContentType == "Banner" && x.Latest).ListAsync();
+
+    foreach (var selectedItem in Model.SelectedItems)
+    {
+        selectedBannerItems.Add(new VueMultiselectBannerItemViewModel
+        {
+            Id = selectedItem.Id,
+            DisplayText = selectedItem.DisplayText,
+            HasPublished = selectedItem.HasPublished,
+            IsActive = bannerContentItems.FirstOrDefault(x => x.ContentItemId == selectedItem.Id).Content.Banner.IsActive.Value,
```
Comment (ravi-methods) [inline DFC.ServiceTaxonomy.ContentPickerPreview/Views/ContentPickerField-Banner.Edit.cshtml:29]: FirstOrDefault could return null?  or any property after.. You may want to add null propagation and default it to false.

Comment (pramireddy) [reply DFC.ServiceTaxonomy.ContentPickerPreview/Views/ContentPickerField-Banner.Edit.cshtml]: Not possible to get null at this stage. But Updates code to handle null

File: DFC.ServiceTaxonomy.ContentPickerPreview/Controllers/BannerContentPickerAdminController.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            ContentPartFieldDefinition? partFieldDefinition = _contentDefinitionManager.GetPartDefinition(part)?.Fields.FirstOrDefault(f => f.Name == field);
+
+            ContentPickerFieldSettings? fieldSettings = partFieldDefinition?.GetSettings<ContentPickerFieldSettings>();
+
+            if (fieldSettings == null)
+            {
+                return BadRequest("Unable to find field definition");
+            }
+
+            string? editor = partFieldDefinition?.GetSettings<ContentPartFieldSettings>().Editor ?? "Default";
```
Comment (ravi-methods) [inline DFC.ServiceTaxonomy.ContentPickerPreview/Controllers/BannerContentPickerAdminController.cs:45]: GetSettings<ContentPartFieldSettings>() can be null as well?

Comment (pramireddy) [reply DFC.ServiceTaxonomy.ContentPickerPreview/Controllers/BannerContentPickerAdminController.cs]: Updated code

**Review by ravi-methods — APPROVED** [review]

**Pull request merged**